src/ghdcbot/engine/pr_status.py

```
@@ -0,0 +1,541 @@
+    # CodeRabbit inline review comments & unresolved review threads
+    coderabbit_count = 0
+    if not coderabbit_approved:
+        # First check GraphQL review threads (if supported and returns a non-empty list of dicts)
+        get_threads = getattr(github_adapter, "get_pull_request_review_threads", None)
+        raw_threads = get_threads(owner, repo, pr_number) if callable(get_threads) else None
+        if isinstance(raw_threads, list) and raw_threads:
+            for t in raw_threads:
+                if isinstance(t, dict):
+                    authors = t.get("authors") or []
+                    if any(a in bot_logins_lower for a in authors):
+                        if not t.get("is_resolved") and not t.get("is_outdated"):
```

Ruff (0.16.1)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)